docs/docs/reference/shards/UI/index.md

```diff
@@ -0,0 +1,9 @@
+# UI
```

Comment:
That file shouldn't exist since we generate it for `(UI)` aka the context.

Reply:
I don't think we should have it as a top level page if UI is a shard itself and has children.
I let it generate in the default path (General/UI.md) and linked to there since that avoids having to manually copy it